client/src/pages/Eesha.jsx

```diff
@@ -1,14 +1,181 @@
+          <p className="eesha-sign-chart-h1">venus || scorpio || 10°20'31"</p>
+          <p className="eesha-sign-chart-h2">my romantic side is mysterious, passionate, and seductive..</p>
+          <p className="eesha-sign-chart-h2">aaaand that's enough of that.</p>
+        </div>
+
+        <div className="grid-six">
+          <img src={mar} alt="mar-sign" />
+          <p className="eesha-sign-chart-h1">mars || scorpio || 9°22'46"</p>
+          <p className="eesha-sign-chart-h2">i assert myself in a way that is serious and incisive.</p>
+          <p className="eesha-sign-chart-h2">once i decide i want to do something, i don't hold back.</p>
+        </div>
+
+        <div className="grid-seven">
+          <img src={jup} alt="jup-sign" />
+          <p className="eesha-sign-chart-h1">jupiter || leo || 17°52'53"</p>
+          <p className="eesha-sign-chart-h2">i grow and find understanding through magnanimity, inspiring confidence, thinking big, and feeling good about myself.</p>
+        </div>
+
+        <div className="grid-eight">
+          <img src={sat} alt="sat-sign" />
+          <p className="eesha-sign-chart-h1">saturn || gemini || 25°43'34"</p>
+          <p className="eesha-sign-chart-h2">i struggle with a lack of focus, restless energy, and getting bored easily.</p>
+          <p className="eesha-sign-chart-h2">no i don't.</p>
+        </div>
+
+        <div className="grid-nine">
+          <img src={ur} alt="ur-sign" />
+          <p className="eesha-sign-chart-h1">uranus || aquarius || 25°38'59"</p>
+          <p className="eesha-sign-chart-h2">other generations are shocked by our generation's unconventionality, intellectuality, and detachedness.</p>
+          <p className="eesha-sign-chart-h2">is this true, h4i?</p>
+        </div>
+      </div>
+
+      <div className="callout">
+        <div className="callout-header">my celtic spread</div>
+        <div className="callout-container">
+          <p className="eesha-text-unpadded">from my latest reading </p>
+        </div>
+      </div>
+
+      <div className="flex-grid-container-cards">
+        <div className="grid-4">
+          <img src={eightwands} alt="8 of wands" />
+        </div>
+
+        <div className="grid-6">
+          <img src={chariot} alt="the chariot" />
+        </div>
+
+        <div className="grid-8">
+          <img src={fourswords} alt="4 of swords" />
+        </div>
+
+        <div className="grid-9">
+          <img src={sixwands} alt="6 of wands" />
+        </div>
+
+        <div className="grid-10">
+          <img src={blocked} alt="blocked" />
+        </div>
+
+        <div className="grid-11">
+          <img src={fourcups} alt="4 of cups" />
+        </div>
+
+        <div className="grid-12">
+          <img src={fiveswords} alt="5 of swords" />
+        </div>
+
+        <div className="grid-14">
+          <img src={lovers} alt="lovers" />
+        </div>
+
+        <div className="grid-16">
```

Comment:
the layout of these are showing up interestingly on my screen: 
![image](https://user-images.githubusercontent.com/34987983/138349487-79ad2d6d-0755-4c64-b986-9ac8fbe42080.png)